packages/components/src/timeline/index.js

```diff
@@ -0,0 +1,7 @@
+const Timeline = () => {
+	return (
+		<div>Hello world.</div>
```

Comment:
This string needs to be passed through the translation method. :trollface: I jest - excited to see this contribution getting started and happy to help in any way @v18

Reply:
LOL. thank you 🙏